source/client-side-encryption/corpus/corpus-encrypted.json

```diff
@@ -7681,5 +7681,1835 @@
     "value": {
       "$maxKey": 1
     }
+  },
```

Comment:
corpus-encrypted.json was generated from the C driver PoC after running the updated corpus test.